plugins/kubelet_logs.yaml

```diff
@@ -0,0 +1,60 @@
+version: 0.0.1
+title: Kubernetes Kubelet
+description: Log parser for Kubelet journald logs
+parameters:
+  - name: journald_directory
+    description: Directory containing journal files to read entries from
+    type: string
+  - name: start_at
+    type: string
+    description: At startup, where to start reading logs from the file.
+    supported:
+      - beginning
+      - end
+    default: end
+
+template: |
+  receivers:
+    journald:
+      {{ if .journald_directory }}
+      directory: {{ .journald_directory }}
+      {{ end }}
+      start_at: {{ .start_at }}
+      units:
+        - kubelet
+      operators:
+          # Semantic conventions says node name should be a resource.
+          - type: move
+            from: body._HOSTNAME
+            to: resource["k8s.node.name"]
+          # Replace journald body with application's log message
+          - type: move
+            from: body.MESSAGE
+            to: body
+          # Parse kubelet klog formatted message
+          - type: regex_parser
+            regex: '(?P<severity>\w)(?P<timestamp>\d{4} \d{2}:\d{2}:\d{2}.\d+)\s+(?P<pid>\d+)\s+(?P<src>[^:]*):(?P<src_line>[^\]]*)\] (?P<message>.*)'
+            severity:
+              parse_from: attributes.severity
+              mapping:
+                debug: d
+                info: i
+                warning: w
+                error: e
+                critical: c
+            timestamp:
+              parse_from: attributes.timestamp
+              layout: '%m%d %H:%M:%S.%s'
```

Comment:
This timestamp doesn't have any time offset, I think we might want to add a timezone param (unless we think it's fixed to UTC, in which case we should hardcode to that).

Reply:
@jsirianni Do you happen to know if it always uses UTC?

Reply:
Hmm yeah. Generally it will be UTC but we have seen other timezones in use. This would be the timezone set on the host operating system.

Reply:
I can add it, defaulting to UTC